views/pages/login.ejs

```diff
@@ -1,7 +1,21 @@
 <!doctype html>
 <html lang="en">
 <% include ../includes/head.ejs %>
-
+<style>
+  /* Hack of Bulma CSS lib */
```

Comment:
the thing I am looking for in such comments is the « why ? ».

ex:

/* fix bulma wrong spacing on xxx item */